packages/design-system-react/src/components/AvatarAccount/AvatarAccount.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
```

Comment:
This has been moved to a separate PR https://github.com/MetaMask/metamask-design-system/pull/673